hands-on-exercise/3-cosmjs-adv/4-cosmjs-gui.md

```diff
@@ -258,12 +258,12 @@ For instance, prepare access to `rpcUrl` for `MenuContainer.tsx` by adding it to
     -  const App = () => {
     +  const App = ({ rpcUrl }: AppProps) => {
             ...
-            return (
-                <Container style={styles}>
+                <Routes>
+    -                <Route path="menu" element={<MenuContainer location={""} />} />
     +              <Route path="menu" element={<MenuContainer location={""} rpcUrl={rpcUrl} />} />
```

Comment:
```suggestion
    -              <Route path="menu" element={<MenuContainer location={""} />} />
    +              <Route path="menu" element={<MenuContainer location={""} rpcUrl={rpcUrl} />} />
```
Because of how the presentation of a diff works, the number of spaces on a `-` or `+` lines is a bit different.
You can find out what it looks like when you do `npm run serve`.